app/components/certificate/request.tsx

```diff
@@ -0,0 +1,21 @@
+import { Flex, Heading, Text, HStack } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  cert: Certificate;
+}
+
+export default function Request(props: Props) {
```

Comment:
Sam thing on props for component naming

Reply:
Also name itself seems to be very general, especially the work `Request` Maybe do something like `CertificateRequestView`